src/js/components/SkipLinks.js

```diff
@@ -0,0 +1,75 @@
+// (C) Copyright 2014-2015 Hewlett-Packard Development Company, L.P.
+
+var React = require('react');
+var Layer = require('./Layer');
+var Menu = require('./Menu');
+var DOM = require('../utils/DOM');
+var IntlMixin = require('../mixins/GrommetIntlMixin');
+
+var SkipLinks = React.createClass({
+
+  mixins: [IntlMixin],
+
+  componentDidMount: function () {
+    var anchorElements = document.querySelectorAll('[data-skip-label]');
```

Comment:
Instead of trying to globally grab the document object, which isn' there in server side rendering situations can this be uploaded to use the virtual DOM in React.

For example you can get the virtual document here with...
```js
var document = React.findDOMNode(this).ownerDocument;
```